src/main/java/com/denizenscript/clientizen/scripts/containers/gui/elements/ItemElement.java

```diff
@@ -0,0 +1,48 @@
+package com.denizenscript.clientizen.scripts.containers.gui.elements;
+
+import com.denizenscript.clientizen.objects.ItemTag;
+import com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.YamlConfiguration;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import io.github.cottonmc.cotton.gui.widget.WItem;
+import io.github.cottonmc.cotton.gui.widget.WWidget;
+
+import java.util.List;
+
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getTaggedObject;
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getTaggedObjectList;
+
+public class ItemElement implements GuiScriptContainer.GuiElementParser {
+
+    // <--[language]
+    // @name Item GUI Element
+    // @group GUI System
+    // @description
+    // Item GUI elements show an item, potentially looping over a list of items and switching between them; they have a UI type of "item".
+    //
+    // <code>
+    // ui_type: item
+    // # The item (or list of items) to display, required.
+    // items: <ListTag(ItemTag)>
+    // # The duration to show each item for when more than one is specified, optional.
+    // duration: <DurationTag>
```

Comment:
perhaps `interval` (a certain meaning implies repetition of some kind)

Reply:
Feels a bit confusing imo - `duration` is a known and recognized concept within Denizen, and I don't think it's worth the rename even though it's technically more accurate.
Could maybe clarify it a bit if you think that's needed, e.g. `frame_duration` or something?

Reply:
frame_duration makes sense

Reply:
Renamed